backend/mark_marker/src/main/scala/mark_marker/trainer/DatabaseConnector.scala

```diff
@@ -0,0 +1,96 @@
+package mark_marker.trainer
+
+import java.io._
+import java.sql.DriverManager
+import java.sql.Connection
+
+import edu.stanford.nlp.classify.{Classifier, ColumnDataClassifier}
+
+import scala.io.Source
+
```

Comment:
That would make the entire backend depend on infrastructure.
A correct approach would be to move it where it belongs, in lib and both of us just take it from there.